src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,46 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: $card-gap;
+
+  &__card {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    @include card($card-height, $card-width);
+    border: 1px solid $color-border;
+    border-radius: 5px;
+    color: $color-primary;
+    transition: background-color $duration, color $duration;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $color-primary;
+      color: $color-secondary;
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      @include column(4, $card-width);
+    }
+  }
```

Tip: media queries can be nested in scss. Suggested change will work only if you move this fragment inside `.catalog`.
```suggestion
  @media (min-width: 1024px) {
      @include column(4, $card-width);
  }
```